core/src/main/kotlin/de/fraunhofer/iem/spha/core/strategy/KpiCalculationStrategy.kt

```diff
@@ -60,6 +61,8 @@ internal interface KpiCalculationStrategy {
     fun isValid(node: KpiNode, strict: Boolean = false): Boolean
 }
 
```

Comment:
Is this intented to be on global scope? How does the private modifier work in that case?

Reply:
the logging library says to use it this way, I didn't check why and trusted them. 
private makes it accessible in the whole file and it should be initialized first.